webhooks.md

```diff
@@ -0,0 +1,111 @@
+# Webhooks
+
+Currently supported webhooks for operation on integrations. Action is triggered everytime when hotel updates the integration.
+
+* [Integration created](webhooks.md#integration-created) - Whenever integration is created.
```

Comment:
These links are broken. For header `### Integration created data` the link should be `webhooks.md#integration-created-data`